lizrd/core/llm.py

```diff
@@ -399,16 +478,16 @@ def forward(self, x):
         return embeddings
 
 
-@ash.check("... -> ... d")
-def EmbeddingLayer(*layers):
-    return Sum(*layers)
+class EmbeddingLayer(Aggregate):
+    def __init__(self, *layers):
+        super(EmbeddingLayer, self).__init__((lambda x, y: x + y), *layers)
```

Comment:
this is some Aggregate layer magic with the lambda. Can you describe what it does?